.travis.yml

```diff
@@ -5,11 +5,11 @@ cache:
     - $HOME/.cache/pip
 language: python
 python:
-  - "3.5"
+  - "3.6"
 env:
-  - DJANGO=django~=2.0
-  - DJANGO=django~=2.1
-  - DJANGO=django~=2.2
+  - DJANGO=Django~=2.2
```

Comment:
If using `~=`, please add more specific versions or remove one of the two below as they are effectively the same. 💌

Reply:
Yes, that was an _excellent_ catch :) 

I have to say that behavior violated the [Principle of Least Astonishment](https://en.wikipedia.org/wiki/Principle_of_least_astonishment) for me.